pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -198,6 +198,7 @@ func validatePluginConfig(path *field.Path, apiVersion string, profile *config.K
 
 	if profile.Plugins != nil {
 		stagesToPluginSet := map[string]config.PluginSet{
+			"enqueue":    profile.Plugins.PreEnqueue,
```

Comment:
update to match the new field name
```suggestion
			"preEnqueue":    profile.Plugins.PreEnqueue,
```

Reply:
Updated, and with more literal replacements.